### ISSUE TITLE
Documentation Page: Fix the footer section on cheat sheats page

### ISSUE THREAD

**Issue by mahakbansal2019** [ocaml/v2.ocaml.org#1361]
Footer  should be at the bottom of the page like other pages

![Screenshot from 2021-04-01 22-25-11](https://user-images.githubusercontent.com/53010252/113328243-a5434400-9339-11eb-9129-62cb7f1cb3f3.png)

@patricoferris What do you think about this?
If It's Okay I would like to work on this.

**Comment by patricoferris** [comment]
@mahakbansal2019 yep that would be a great thing to fix thank you, I think this impacts any page that doesn't have particularly long main content section.

**Comment by mahakbansal2019** [comment]
Yeah, that's true @patricoferris!
I'll start working on this and also try to find others who have the same problem.

**Comment by nibble0101** [comment]
@mahakbansal2019 I am just curios. On what screen size is that?

**Comment by mahakbansal2019** [comment]
@nibble0101 screen having witdth >= 768px

**Comment by nibble0101** [comment]
@mahakbansal2019 I have failed to reproduce it on the live site on my  laptop. Did you discover the issue from the live site or the local setup on your machine? I believe that effect has a lot more to do with the screen height than the width.

**Comment by mahakbansal2019** [comment]
@nibble0101  It's on the live site. What do you see on the live site ?

**Comment by nibble0101** [comment]
@mahakbansal2019  I see the page displayed normally on chrome. I suspect  my screen size is not of appropriate height to reproduce the issue or it could be browser related.  Right now I can't emulate larger screen sizes.  The page appears normally like in the screenshot below.

![image](https://user-images.githubusercontent.com/52580190/113337471-7bd1ea80-9330-11eb-91a6-5a08465d6398.png)

**Comment by mahakbansal2019** [comment]
@nibble0101 that's strange.
I'm also on chrome and My max screen size is 1607x949 and it looks weird on that.
@patricoferris Should I go ahead with this?

**Comment by patricoferris** [comment]
I opened chrome dev tools and uses the `responsive` tool and seemed to recreate the issue. I'm also not aware of anything in the code either for clamping the footer to the bottom (although that's from memory I haven't looked), so I think this is a bug.

**Comment by nibble0101** [comment]
@patricoferris  Thanks. I have managed to reproduce it using Chrome devtool  by emulating screen size of  `1024 x 1366 px` - dimensions of ipad pro. I believe one possible solution is to give the container preceding the `footer` a styling of `min-height: 90vh;` or thereabouts.  Assuming there is a container across all pages with a common selector preceding  the `footer`.

**Comment by mahakbansal2019** [comment]
Thanks @nibble0101 for the suggestions :)

**Comment by mahakbansal2019** [comment]
After running `make`, I can see the `cheat_sheets.html` file inside the docs folder. I want to add a div wrapping the content of the body.
But I'm a little curious about how can I add div in the `site/docs/cheat_sheets.md` file so that after running `make` the changes get reflected in the `cheat_sheets.html `file.

![Screenshot from 2021-04-02 13-23-57](https://user-images.githubusercontent.com/53010252/113394976-cb5af980-93b6-11eb-9e57-c9f589b76394.png)


@patricoferris Can you guide me on this?

**Comment by patricoferris** [comment]
So there are lots of ways to fix this. The way you are suggesting could work, you will have to write HTML into the markdown file (markdown supports this), if you have a look at other markdown files (like the main `index.md`) you should be able to see what I mean.

As @nibble0101 suggests you could also look at styling the `div.container` differently too (this impacts every single page). The scripts take individual `md` files and generate `html` and then place them inside that `div`. Have a look at the `templates` directory to hopefully see what I mean.

**Comment by mahakbansal2019** [comment]
I tried adding a `min-height `of around `80vh`. It looked perfect on every page except the home page of the OCaml site. It happens because this page uses the `container` class two times.

I mean I can't modify the css of `container` class so should I go ahead with changing site/docs/cheat_sheets.md and writing HTML inside?
Also Like I said this file has no HTML elements right now, So I need to just see `cheat_sheets.html `and copy-paste the whole code in `cheat_sheets.md` with some changes.
@patricoferris Is this the right approach ? 

Home page:
![Screenshot from 2021-04-02 15-30-22](https://user-images.githubusercontent.com/53010252/113406351-f6e6df80-93c8-11eb-957b-c74af781a042.png)

**Comment by mahakbansal2019** [comment]
@patricoferris I have submitted a PR. 
Please let me know if you need any changes :)

**Comment by mahakbansal2019** [comment]
@patricoferris Could you please guide me on what is wrong with this PR 
https://github.com/ocaml/ocaml.org/pull/1463
